### PR TITLE
MPU+QMC Details, BMI Fix, and Defines.h Generator Update

### PR DESCRIPTION
**Pull request by NWBx01**
Added MPU6050+QMC5883L tracker schematic, as well as relevant instructions for flashing the experimental firmware it requires.

The BMI160 schematic previously had an extraneous detail which has been corrected.

The defines.h generator has been updated to allow selecting the BMI160 and ICM20948 as options. The note that the MPU9250 is experimental has been removed and replaced with a notice about requiring a stable magnetic environment. Likewise, I attempted to make a similar warning for the ICM20948 which notes that to run it in 9DoF mode requires changing a line in debug.h, with another warning about requiring a stable magnetic environment. I'm slightly unsure of whether the ICM20948 warning is functional, but I think it should work.

### CODE REVIEW

File: assets/js/schematics.js [hunk 1]
```diff
@@ -7,14 +7,15 @@
             'options': [
                 [
                     'imu',
-                    (imu) => imu == 'bno' ? 'd1_BNO-Main.png' : (imu == 'mpu' ? 'd1_MPU-Main.png' : (imu == 'mpu9250' ? 'd1_MPU9250-Main.png' : 'd1_BMI160-Main.png'))
-                    
+                    (imu) => imu == 'bno' ? 'd1_BNO-Main.png' : (imu == 'mpu' ? 'd1_MPU-Main.png' : (imu == 'mpu9250' ? 'd1_MPU9250-Main.png' : (imu == 'bmi' ? 'd1_BMI160-Main.png' : 'd1_MPU+QMC-Main.png')))
+                    // I don't think this is how this should be done at all...
```
Comment (calliePepper) [inline assets/js/schematics.js:11]: You aren't wrong here

**Comment by calliePepper** [comment]
Just double checking, you have done a local check on the image / code changes and everything looks peachy?

File: assets/js/schematics.js (hunk 1 above)
Comment (carl-anders) [reply assets/js/schematics.js:11]: It's a function. It's okay to not have it in a single line. It's grown way past that, a inline if was okay for two options, not when it expanded to three, and changing to four now looks quite silly ^^.

It's well time to change it to a function that contains a switch or a little inline look up table maybe? :)

**Review by calliePepper — APPROVED** [review]
There's some solid work here and it looks like you have done a bunch of fixes, great work

**Pull request merged**